composer.json

```diff
@@ -17,7 +17,12 @@
         "php":                      ">=5.3.9",
         "Org_Heigl/Hyphenator":     "~2.0",
         "twig/twig":                "~1.20|~2.0",
-        "symfony/framework-bundle": "~2.0|~3.0"
+        "symfony/framework-bundle": "~2.3|~3.0"
```

Comment:
minimum requirement is now 2.3 .. 2. would probably be possible but I had some issues getting the tests to run and then decided to not bother with it ..